src/cbc_sdk/platform/asset_groups.py

```diff
@@ -94,6 +97,104 @@ def _query_implementation(cls, cb, **kwargs):
         """
         return AssetGroupQuery(cls, cb)
 
+    def list_member_ids(self, **kwargs):
```

Comment:
Why kwargs instead of parameters with default values? Also the name filter is confusing and requires you to read the description would recommend `membership`

```
def list_member_ids(self, start=0, rows=20, membership="ALL"):
```